mdx.opam

```diff
@@ -14,7 +14,7 @@ build: [
 ]
 
 depends: [
-  "ocaml" {>= "4.02.3"}
+  "ocaml" {>= "4.02.3" & < "4.10.0"}
```

Comment:
Can you remove the `< 4.10.0` bound? There's nothing in mdx preventing it from working with 4.10. As soon as odoc gets a 4.10 compatible release we will be able to build there as well!

Reply:
True! Done